sources/packages/api/src-sql/Restrictions/Create-restrictions.sql

```diff
@@ -1,20 +1,22 @@
 -- Create Table sims.restrictions
 CREATE TABLE IF NOT EXISTS sims.restrictions(
   id SERIAL PRIMARY KEY,
-  restriction_type sims.restriction_types NOT NULL, 
+  restriction_type sims.restriction_types NOT NULL,
```

Comment:
We should not modify the existing restrictions files. Am I missing something? Why this approach was taken?

Reply:
no, I didn't modify anything, it's the formatting changes.. earlier I think we didn't have.SQL formatter

Reply:
So, there are no changes required here, right? Can we please remove this file from the PR?

Reply:
I added the file because of formatting..ya i can remove

Reply:
I also feel, it is the formatter